### PR TITLE
NHT<>USDT Auction Dca

### PR DESCRIPTION
**Pull request by Siddharth2207**
## Motivation
NHT<>USDT Auction Dca

## Checks

By submitting this for review, I'm confirming I've done the following:
- [x] made this PR as small as possible
- [x] unit-tested any new functionality
- [x] linked any relevant issues or PRs
- [x] included screenshots (if this involves a front-end change)

### CODE REVIEW

**Comment by Siddharth2207** [comment]
- Buy NHT : 
image

- Sell NHT : 
image

File: public/_strategies/raindex/1-auction-dca/auction-dca.rain [hunk 1]
```diff
@@ -180,6 +180,14 @@ tokens:
     network: polygon
     address: 0x3c499c542cEF5E3811e1192ce70d8cC03d5c3359
     decimals: 6
+  polygon-usdt:
```
Comment (thedavidmeister) [inline public/_strategies/raindex/1-auction-dca/auction-dca.rain:183]: issue title says usdc but this is usdt

**Review by thedavidmeister — CHANGES_REQUESTED** [review]

File: public/_strategies/raindex/1-auction-dca/auction-dca.rain (hunk 1 above)
Comment (Siddharth2207) [reply public/_strategies/raindex/1-auction-dca/auction-dca.rain:183]: Edit, its USDT. The pool is NHT-USDT : 
https://www.dextools.io/app/en/polygon/pair-explorer/0x356a8af4b85cebf8cb6a0331fffc562bce9967d7?t=1734075520601

**Review by hardyjosh — APPROVED** [review]

**Comment by Siddharth2207** [comment]
@hardyjosh Im closing this and starting a fresh PR

**Pull request closed without merging**